nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -353,7 +386,44 @@ def validation_step(self, batch, batch_idx):
         if loss_mean.item == 0.0:
             loss_mean = []
 
-        return loss_mean
+        if self.cfg.get('report_validation_metric', False):
+            preds_text, labels_text = [], []
+            input_ids, labels, loss_mask, position_ids, attention_mask, taskname_ids = batch
+            input_lenghts = torch.argmax(loss_mask, 1, keepdim=True)
```

Comment:
nit: can we change input_lenghts --> input_lengths. Technically both are ok but trying to be consistent with self.length_params